pkgs/data/fonts/sarasa-gothic/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, p7zip}:
+
+stdenv.mkDerivation (rec {
+  version = "0.5.2";
+  name = "sarasa-gothic-${version}";
+
+  package = fetchurl {
+    url = "https://github.com/be5invis/Sarasa-Gothic/releases/download/v${version}/sarasa-gothic-ttf-${version}.7z";
+    sha256 = "18ycw57k7yhrbb8njzhzk6x32xnjal61wr48qxkqy4lh9zfy0p22";
+  };
+
+  nativeBuildInputs = [ p7zip ];
+
+  unpackPhase = ''
+    7z x $package
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp *.ttf $out/share/fonts/truetype
+  '';
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  meta = {
```

Comment:
To avoid having to prefix `stdenv.lib` to the lines below, you can use `meta = with stdenv.lib; {` here instead.